log/keyvalue.go

```diff
@@ -300,6 +301,38 @@ func (v Value) String() string {
 	}
 }
 
+// MarshalJSON implements a custom marshal function to encode Value.
+func (v Value) MarshalJSON() ([]byte, error) {
+	var jsonVal struct {
+		Type  string
+		Value interface{}
```

Comment:
Cannot we just marshal the value (without the type)?

I guess this is for having the same experience as for `stdouttrace` and `stdoutmetric`. Personally, I do not like it and I would change it there as well.

Probably it would be better to handle it as a separate PR(s) for all stdout exporters.

Reply:
I know having a type is kind of annoying, but this is more like a debug message. It might be better to have a type so users know the concrete value.

For instance, a value can be a byte with `[]byte{5}`, it can also be a string with `BQ==`. The output of these two values will be the same without the type.